### PR TITLE
Add material to chapter on technology introductions

### PR DESCRIPTION
**Pull request by ErinBecker**
This still has some clean-up needed, but is ready for initial review @fmichonneau

### CODE REVIEW

File: 05-technological-introductions.Rmd [hunk 1]
```diff
@@ -1,10 +1,399 @@
...
+1. A few paragraphs of explanatory text describing the lesson. 
+1. One or more `.prereq` boxes detailing the lesson's prerequisites, giving an overview of the dataset, and/or calling attention to the lesson's Instructor notes.
+
+The schedule will automatically be included in the lesson homepage based on information present in the 
+episode files. 
+
+### Episode files
+
+The majority of a lessons content is in its episode files. 
+Episode files are stored in the `_episodes/` folder within your lesson repo.
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: should we mention the `_episodes_rmd/` here as well?

File: 05-technological-introductions.Rmd [hunk 2]
```diff
@@ -1,10 +1,399 @@
...
+episode files. 
+
+### Episode files
+
+The majority of a lessons content is in its episode files. 
+Episode files are stored in the `_episodes/` folder within your lesson repo. 
+Episode file names must start with a two-digit identifier number (e.g. `01`) followed
+by a short descriptive name, separated by a dash (`-`). For example `02-loop.md`, `03-lists.md`. 
+The numeric identifier is used to place your episode files in the correct sequence within the lesson. 
+Episode files are written in *Markdown* (more on that in a moment).
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: (or Rmarkdown)

File: 05-technological-introductions.Rmd [hunk 3]
```diff
@@ -1,10 +1,399 @@
...
+*   lists
+*   tables
+*   images or figures
+*   code chunks
+*   special blockquotes, including exercises and solutions (described below)
+
+This content will be written in *Markdown*, a light-weight markup language that makes it possible to create
+fancy HTML pages using only a few formating tricks. In this section, we'll cover only the Markdown syntax that
+you will need in order to create the content types listed above. You can find more information about Markdown
+at ...
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: maybe this one: https://commonmark.org/help/

File: 05-technological-introductions.Rmd [hunk 4]
```diff
@@ -1,10 +1,399 @@
...
+| Food     | Sandwich |
+| Drink    | Tea | 
+| Food     | Apple |
+
+
+1. Images or figures - Place a copy of the image you would like to display into the `fig` directory. You can
+then link to the figure using the syntax:
+
+```
+![Figure Description](../fig/figure_file_name.svg)]
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: there is an `]` at the end of the line

File: 05-technological-introductions.Rmd [hunk 5]
```diff
@@ -1,10 +1,399 @@
...
+
+All Carpentries lesson materials, whether they are established or in early development, are hosted publically 
+on GitHub in one of several *organisations*. The following high-level organisations are managed by The Carpentries:
+
+* [carpentries](https://github.com/carpentries) - hosts The Carpentries website and materials for programs that span individual lesson programs, such as our Instructor Training curriculum and The Carpentries Handbook
+* [datacarpentry](https://github.com/datacarpentry) - hosts Data Carpentry specific lesson materials and website
+* [librarycarpentry](https://github.com/librarycarpentry) - hosts Library Carpentry specific lesson materials and website
+* [swcarpentry](https://github.com/swcarpentry) - hosts Software Carpentry specific lesson materials and website
+* [data-lessons](https://github.com/data-lessons) - hosts lessons in development which are targeted to become part of the official Carpentries lesson stack
+* [carpentrieslab](http://github.com/carpentrieslab) - hosts lessons in development which may or may not become part of the official Carpentries lesson stack
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: for Carpentries lab maybe add: "hosts community-developed lessons which may or may not become part of the official Carpentries lesson stack".

File: 05-technological-introductions.Rmd [hunk 6]
```diff
@@ -1,10 +1,399 @@
...
+
+In order to contribute to lesson materials, you will need a GitHub ID.
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: maybe replace ID with account

File: 05-technological-introductions.Rmd [hunk 7]
```diff
@@ -1,10 +1,399 @@
...
+
+In order to contribute to lesson materials, you will need a GitHub ID. 
+
+To manage changes, we follow [GitHub flow][github-flow]. 
+Each lesson has two maintainers who review issues and pull requests or encourage others to do so.
+The maintainers are community volunteers and have final say over what gets merged into the lesson.
+To use the web interface for contributing to a lesson:
+
+1.  Fork the originating repository to your GitHub profile.
+2.  Within your version of the forked repository, move to the `gh-pages` branch and
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: maybe change `gh-pages` to "default branch (e.g., `gh-pages`)". Depending on the setup, we're having several lessons that are not using `gh-pages` as the default branch anymore.

File: 05-technological-introductions.Rmd [hunk 8]
```diff
@@ -1,10 +1,399 @@
...
+* [carpentrieslab](http://github.com/carpentrieslab) - hosts lessons in development which may or may not become part of the official Carpentries lesson stack
+
+In order to contribute to lesson materials, you will need a GitHub ID. 
+
+To manage changes, we follow [GitHub flow][github-flow]. 
+Each lesson has two maintainers who review issues and pull requests or encourage others to do so.
+The maintainers are community volunteers and have final say over what gets merged into the lesson.
+To use the web interface for contributing to a lesson:
+
+1.  Fork the originating repository to your GitHub profile.
```
Comment (fmichonneau) [inline 05-technological-introductions.Rmd]: replace profile with account

**Comment by ErinBecker** [comment]
Thanks for the review @fmichonneau! I've made changes based on your comments and will merge this PR once checks have passed.

**Pull request merged**